apps/src/templates/ReCaptchaDialog.story.jsx

```diff
@@ -1,23 +1,21 @@
 import React from 'react';
 import ReCaptchaDialog from './ReCaptchaDialog';
 
+export default {
+  title: 'ReCaptchaDialog',
+  component: ReCaptchaDialog
+};
+
 //Using Google's publicly available test key:
 //https://developers.google.com/recaptcha/docs/faq#id-like-to-run-automated-tests-with-recaptcha.-what-should-i-do
-export default storybook => {
-  return storybook.storiesOf('Dialogs/ReCaptchaDialog', module).addStoryTable([
-    {
-      name: 'Dialog open',
-      description:
-        'Dialog forces user to cancel or complete captcha before submitting',
-      story: () => (
-        <ReCaptchaDialog
-          siteKey="6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI"
-          handleSubmit={() => {}}
-          handleCancel={() => {}}
-          submitText="Join section"
-          isOpen={true}
-        />
-      )
-    }
-  ]);
+const DEFAULT_PROPS = {
+  handleSubmit: () => {},
+  handleCancel: () => {},
+  isOpen: true,
+  submitText: 'Join section',
+  siteKey: '6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI'
 };
+const Template = args => <ReCaptchaDialog {...DEFAULT_PROPS} {...args} />;
+
+export const DialogOpen = Template.bind({});
+DialogOpen.args = {};
```

Comment:
Are you assigning this empty object to `args` for readability and consistency? (Even though there are no args being passed in?). I had something similar, and opted to leave this line out, but now seeing it I kind of wish I left it in to keep things consistent.

Reply:
I left it their due to my own ignorance. I didn't even try leaving it out. I think you make a good point about keeping it consistent. I will try leaving it there and see how I feel about it over time.